lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -356,7 +356,7 @@ e2function void entity:propSetElasticity(number elasticity)
 end
 
 e2function number entity:propGetElasticity()
-	if not PropCore.ValidAction(self, this, "elasticity") then return end
+	if not PropCore.ValidAction(self, this, "elasticity") then return 0 end
```

Comment:
Not sure if we should return 0 as the wiki doesn't indicate what range of values `GetElasticity` can return.